### DISCUSSION TITLE
Python errors on AWS METviewer server

### DISCUSSION THREAD

**Discussion by LoganDawson-NOAA** [dtcenter/METplus#969] [Plot Generation] [answered]
I attempted making a time series plot on [AWS METviewer](metviewer.nws.noaa.gov) using Python execution, but the plotting failed with a series of Python errors. There are some outdated package warnings that _may_ not be fatal. The actual fatal error appears to be related to sorting data values.

XML script and the log file are attached.
[plot_20210628_142418.xml.txt](https://github.com/dtcenter/METplus/files/6733619/plot_20210628_142418.xml.txt)
[plot_20210628_142418.log.txt](https://github.com/dtcenter/METplus/files/6733620/plot_20210628_142418.log.txt)

I will note that this plot was generated successfully using Rscript execution, so the plotting should work with Python.

**Comment by jprestop** [comment]
Hi Logan.  Thanks for letting us know that you are having trouble with AWS METviewer.  @TatianaBurek could you please take a look and help Logan?  @TatianaBurek, please also review the labels I added and add/delete any appropriate labels.

**Reply by jprestop** [reply]
@LoganDawson-NOAA, @TatianaBurek has Jury Duty this week.  Unfortunately, she will not be able to respond until next week.

**Reply by LoganDawson-NOAA** [reply]
@jprestop, not a problem - it's not urgent at all. I wanted to go ahead and submit the question before it slipped my mind.

**Comment by TatianaBurek** [comment, marked as answer]
@LoganDawson-NOAA 
Logan,
I fixed the issue and it should work on AWS:
20210701_195725
Looks like the data has some gaps so I selected NOT to connect across NA for all series.
Let me know if it works for you

**Comment by TatianaBurek** [comment]
The issue was created:
https://github.com/dtcenter/METplotpy/issues/129
